src/otaclient_common/downloader.py

```diff
@@ -52,6 +52,9 @@
 logger = logging.getLogger(__name__)
 
 EMPTY_FILE_SHA256 = r"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+EMPTY_FILE_SHA256_BYTE = bytes.fromhex(
+    "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+)
```

Comment:
This is same as `orm.EMPTY_FILE_SHA256_BYTE`.
Can we use the above in `_delta_gen.py` instead of the duplicated definitions?

Reply:
Actually this value should not bound to detailed module, let me move it under the otaclient_common package as a shared const for the whole otaclient.